deselected_tests.yaml

```diff
@@ -633,3 +633,87 @@ gpu:
   - model_selection/tests/test_validation.py::test_validation_curve_cv_splits_consistency
   - model_selection/tests/test_search.py::test_grid_search_cv_results
   - model_selection/tests/test_search.py::test_random_search_cv_results
+
```

Comment:
Now this would disable tests for both CPU and GPU - we need to change this for GPU only

Reply:
thanks, wasn't scrolling that high.